tox.ini

```diff
@@ -92,9 +110,10 @@ envlist =
 [testenv]
 deps =
   -c dev-requirements.txt
-  opentelemetry: pytest
-  opentelemetry: pytest-benchmark
-  opentelemetry: flaky
+  lint: -r lint-requirements.txt
+  test-opentelemetry: pytest
```

Comment:
We are not renaming the envs, here what we are doing is telling `tox` to install certain packages for `lint` testing and other ones for regular testing. Since `lint` environments are not called `lint-opentelemetry-xyz`, not using `test-opentelemetry` here would lead to unnecessary packages being installed in `lint` environments.